libcaf_test/caf/test/runner.cpp

```diff
@@ -68,7 +67,17 @@ int runner::run(int argc, char** argv) {
     return EXIT_SUCCESS;
   }
   default_reporter->start();
+  auto enabled = [](const std::regex& selected,
+                    const std::string& search_string) {
+    return std::regex_search(search_string, selected);
+  };
+  std::regex suite_regex;
+  if (auto suite_regex_string = get_as<std::string>(cfg_, "suites")) {
+    suite_regex = std::regex(*suite_regex_string);
+  }
```

Comment:
This can be shortened:

```suggestion
  auto suite_regex = std::regex{get_or(cfg_, "suites", ".*")};
```

Reply:
I was planning to add some type of verification of the regex string as regex simply crashes the application if something weird is passed to the application. Something simple like "*" throws exception currently. Let me know your thoughts on this.

Reply:
Good point. I think we can do the verification in a lambda / member function like `optional<regex> parse_regex_opt("suites")` that then internally catches the exception (or calls `abort` when exceptions are off). My thinking is that we will eventually have two regex parameters (when adding `-t`) and doing the verification here would mean repeating the code.